include/net/cloud.h

```diff
@@ -187,6 +187,8 @@ struct cloud_api {
 				size_t list_count);
 	int (*user_data_set)(const struct cloud_backend *const backend,
 			     void *user_data);
+	int (*get_id)(const struct cloud_backend *const backend,
```

Comment:
The other function names in the API have the verb last, so this could be changed to `id_get` to align with that convention